systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java

```diff
@@ -30,17 +26,20 @@
 import io.enmasse.iot.model.v1.IoTProject;
 import io.enmasse.systemtest.CustomLogger;
 import io.enmasse.systemtest.Endpoint;
-import io.enmasse.systemtest.bases.IoTTestBase;
+import io.enmasse.systemtest.bases.IoTTestBaseWithShared;
 import io.enmasse.systemtest.iot.CredentialsRegistryClient;
 import io.enmasse.systemtest.iot.DeviceRegistryClient;
 import io.enmasse.systemtest.iot.HttpAdapterClient;
+import io.enmasse.systemtest.iot.MessageSendTester;
+import io.enmasse.systemtest.iot.MessageSendTester.ConsumerFactory;
+import io.enmasse.systemtest.iot.MessageSendTester.Type;
 import io.enmasse.systemtest.utils.IoTUtils;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
 @Tag(sharedIot)
 @Tag(smoke)
-public abstract class DeviceRegistryTestBase extends IoTTestBase {
+public abstract class DeviceRegistryTestBase extends IoTTestBaseWithShared {
```

Comment:
Because the tests require the `iotAmqpClientFactory` and the IoTProject in order to test the successful message flow. That seems to be available in `IoTTestBaseWithShared` but not in `IoTTestBase`.

Reply:
Ok, I now realized you changed the whole setup, I did adapt to that new setup, and therefore actually can't use the client provided in the base class. So I am back to using `IoTTestBase` now.